src/service/helper.js

```diff
@@ -0,0 +1,34 @@
+import log from "loglevel";
+import handShake from "../get/handshake.js";
+
+let logLevel = process.env.LOG_LEVEL || "DEBUG";
+log.setLevel(log.levels[logLevel]);
+
+// A 30 second wait timer to allow the messages to be received and processed by mesh
+// Override this value by passing custom seconds when invoked
+export async function waitThirtySeconds(override) {
+  log.debug("\nChecking connection to mailbox with handshake");
+  return new Promise((resolve) => {
+    if (override === undefined) {
+      setTimeout(() => {
+        resolve("30 seconds have passed.");
+      }, 30000); // 30000 milliseconds = 30 seconds
```

Comment:
dont need clarifcation for ms. In docs of MDN and your method has specifed seconds so this conversion is assumed by caller of function.

Reply:
To be fair this might be inherited from Andrew's prototype.

Reply:
Can make the timeout an environmental variable